UIforETW/UIforETW.rc

```diff
@@ -298,6 +300,17 @@ END
 
 /////////////////////////////////////////////////////////////////////////////
 //
+// AFX_DIALOG_LAYOUT
+//
+
+IDD_UIFORETW_DIALOG AFX_DIALOG_LAYOUT

```

Comment:
Interesting. Well, no harm - leave it or delete it.

Good to see Microsoft finally supporting sizeable dialogs - I've been implementing them for 20 years without any help, but maybe they'll become more common now.